stages/mvstex.py

```diff
@@ -110,6 +110,15 @@ def add_run(nvm_file, primary=True, band=None):
                         '{nadirMode} '
                         '{labelingFile} '.format(**kwargs))
                 
+                # Backward compatibility: copy odm_textured_model_geo.mtl to odm_textured_model.mtl
+                # for certain older WebODM clients which expect a odm_textured_model.mtl
+                # to be present for visualization
+                # We should remove this at some point in the future
```

Comment:
Do you want me to open an issue for this just so we can stumble on it in 6-12 months? :D

Reply:
:D sure! I'm probably going to forget otherwise.

Reply:
Lol. Sounds good.

Reply:
Yeah! I got issue 1234. That's auspicious: https://github.com/OpenDroneMap/ODM/issues/1234